package/libs/openssl/Makefile

```diff
@@ -122,18 +213,96 @@ ifndef CONFIG_OPENSSL_WITH_EC2M
   OPENSSL_OPTIONS += no-ec2m
 endif
 
-ifndef CONFIG_OPENSSL_WITH_SSL3
-  OPENSSL_OPTIONS += no-ssl3 no-ssl3-method
+ifndef CONFIG_OPENSSL_WITH_ERROR_MESSAGES
+  OPENSSL_OPTIONS += no-err
 endif
 
-ifndef CONFIG_OPENSSL_HARDWARE_SUPPORT
-  OPENSSL_OPTIONS += no-hw
+ifdef CONFIG_OPENSSL_USE_VERSION_1_1_1
+# For reproducible builds, set a fixed date to show. It could be
+# `date -r $(PKG_BUILD_DIR)/README %s`, but 1970-01-01 00:00:01 will get
+# the message through. This may not be needed with the final release.
+# Note that 0 won't work!
+  OPENSSL_MAKEFLAGS += SOURCE_DATE_EPOCH=1
```

Comment:
Can we use the buildroot global `SOURCE_DATE_EPOCH` here?

Reply:
Yes, we can!